lambdatimes/src/components/Content/Content.js

```diff
@@ -31,13 +37,18 @@ export default class Content extends Component {
       
       Notice that we're passing this function to our <Cards /> component below.
       This function returns an array of cards, so we can just pass it down as such.
-
       Your algorithim for the logic here is as follows: 
         - if the selected tab is 'all' it should return all 
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+    return this.state.cards.filter(card => {
```

Comment:
there's a warning in the browser ` Line 46: Expected to return a value at the end of arrow function array-callback-return` You don't have a base `else` statement here. Add a base statement and warning will go away. You should use eslint to save you from tiny mistakes like these.